services/worker/src/worker/dtos.py

```diff
@@ -240,6 +241,7 @@ class CompatibleLibrary(TypedDict):
 class DatasetCompatibleLibrariesResponse(TypedDict):
     tags: list[DatasetTag]
     libraries: list[CompatibleLibrary]
+    formats: list[DatasetFormat]
```

Comment:
having two formats means that two configs are in different formats, right?

Reply:
Currently having multiple formats is not supported (only one is detected)

Reply:
Ok. I was wondering why we return a list of formats